sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2625,9 +2625,8 @@ object GpuOverrides {
         override def convertToGpu(): GpuExpression = GpuPosExplode(childExprs.head.convertToGpu())
       }),
     expr[CollectList](
-      "Collect a list of elements, now only supported by windowing.",
-      // It should be 'fullAgg' eventually but now only support windowing,
-      // so 'aggNotGroupByOrReduction'
+      "Collect a list of non-unique elements, only supported in rolling window in current.",
```

Comment:
I think the original explanation was clearer. If we want to follow what Spark wrote it would be "Collect the values into a list"

Reply:
Fixed.